students/AndrewMiotke/lesson03/assignment/tests/test_gradel03.py

```diff
@@ -0,0 +1,167 @@
+"""
+    Autograde Lesson 3 assignment
+    Run pytest
+    Run cobverage and linitng using standard batch file
+    Student should submit an empty database
+
+"""
+
+import pytest
+
+# import basic_operations as l
+from src import basic_operations as l
+
+@pytest.fixture
+def _add_customers():
+    return [
+        ("123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("456", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("789", "Name", "Lastname", "Address", "phone", "email", "active", 0),
+        ("345", "Name", "Lastname", "Address", "phone", "email", "active", -10),
+        ("0123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("777", "Name", "Lastname", "Address", "phone", "email", "active", 999)
+    ]
+
+@pytest.fixture
+def _search_customers(): # needs to del with database
+    return [
+        [("998", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+         ("997", "Name", "Lastname", "Address", "phone", "email", "inactive", 10)],
+         ("998", "000")
+    ]
+@pytest.fixture
+def _delete_customers(): # needs to del with database
+    return [
+        ("898", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("897", "Name", "Lastname", "Address", "phone", "email", "inactive", 10)
+    ]
+
+@pytest.fixture
+def _update_customer_credit(): # needs to del with database
+    return [
+        ("798", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("797", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("796", "Name", "Lastname", "Address", "phone", "email", "inactive", -99)
+    ]
+
+@pytest.fixture
+def _list_active_customers():
+    return [
+        ("598", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("597", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("596", "Name", "Lastname", "Address", "phone", "email", "inactive", 99),
+        ("595", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("594", "Name", "Lastname", "Address", "phone", "email", "active", 10),
+        ("593", "Name", "Lastname", "Address", "phone", "email", "active", 99)
+    ]
+
+def test_list_active_customers(_list_active_customers):
+    """ actives """
+    for customer in _list_active_customers:
+        l.add_customer(customer[0],
+                       customer[1],
+                       customer[2],
+                       customer[3],
+                       customer[4],
+                       customer[5],
+                       customer[6],
+                       customer[7]
+                       )
+    actives = l.list_active_customers()
+
+    assert actives == 2
```

Comment:
In the _list_active_customers, we have a total of 4 users which are "active" so this should be assert actives == 4